chart/values.yaml

```diff
@@ -446,15 +448,12 @@ opentelemetryOperator:
             exporters: [logging, otlp]
             processors: [batch]
 
-  jaegerPromscaleQuery:
-    image: timescale/jaeger-query-proxy:0.0.1
-    endpoint: "{{ .Release.Name }}-jaeger-promscale.{{ .Release.Namespace }}.svc.cluster.local:16686"
+  jaeger:
+    image: jaegertracing/jaeger-query:1.30
```

Comment:
nit: I might consider making the env variables (SPAN_STORAGE_TYPE=grpc-plugin) also configurable here. But that is a minor point and shouldn't block the PR

Reply:
Updated!